cmd/login.go

```diff
@@ -150,7 +161,7 @@ func runOCMWorkspaceContainer(
 	}
 
 	var config ocmWorkspaceConfig
-	err := viper.Unmarshal(&config)
+	err = viper.Unmarshal(&config)
```

Comment:
You might need to update the config struct to include the backplaneConfigProd and userHome.